cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -101,7 +115,7 @@ import           Cardano.Api.Value
 --
 -- There are also paramaters fixed in the Genesis file. See 'GenesisParameters'.
 --
-data ProtocolParameters =
+data ProtocolParameters era =
```

Comment:
Why did we need the `era` type param? We don't seem to use it below.

If we really don't need it to be era-parameterised, then it should simplify things elsewhere (e.g. in the cli code).

Reply:
This appears to be leftover, I'll remove it

Reply:
Hmm. I've had a look and I don't think that's the case. You're using it to restrict the fields we render to those from a specific era, but I don't see any need to do that. We can just render them all.